src/sqlfluff/rules/L054.py

```diff
@@ -93,7 +93,7 @@ def _eval(self, context: RuleContext) -> Optional[LintResult]:
             return None
 
         # Ignore Windowing clauses
-        if context.functional.parent_stack.select(sp.is_type(*self._ignore_types)):
```

Comment:
In a recent PR, I suggested using `select()`. It works, but since the goal is simply to look for the existence of *at least one* element, using `any()` will often be faster, since it stops on the first match.

Reply:
Let's do this in a separate PR: #2476